src/components.py

```diff
@@ -4,20 +4,24 @@
 
 # Create a _RELEASE constant. We'll set this to False while we're developing
 # the component, and True when we're ready to package and distribute it.
-_RELEASE = True
+_RELEASE = False
 
-if not _RELEASE:
```

Comment:
why does this need to be kept inside the flash_viewer_grid_component function?

Reply:
I remember having issues with the streamlit component being created multiple times when different datasets were selected. However, I could not replicate it just now. Do you see any downsides with it being handled this way?

Reply:
I just see no reason to declare components whenever this function is called, basically whenever new experiment data is used..

Reply:
I agree. I think this might have only been an issue under windows. I will try to replicate the error and if I cannot I will roll back this change.